api/app/src/command/webhook/webhook.ts

```diff
@@ -121,18 +120,24 @@ export const processData = async <T extends MetriportData>(data: UserData<T>[]):
           // now that we have a all the chunks for one customer, process them
           const settings = await getSettingsOrFail({ id: cxId });
           await processOneCustomer(cxId, settings, payloads);
-          await reportUsage(
+          reportUsage(
             cxId,
             dataAndUserList.map(du => du.cxUserId)
           );
         } catch (err) {
           const msg = getErrorMessage(err);
           log(`Failed to process data of customer ${cxId}: ${msg}`);
+          Sentry.captureException(err, {
+            extra: { cxId, context: `webhook.processData.customer` },
+          });
```

Comment:
General design comment:

Should we wrap the `Sentry.x` invocations in generic helper methods so that we can change the underlying implementation in a layer we control?

Similar to how we have `commands` for db calls instead of raw sequelize calls

Reply:
Yup, makes sense.

Reply:
Done 👍